build.gradle

```diff
@@ -750,3 +750,29 @@ class DockerPushTask extends Exec {
         }
     }
 }
+
+apply from: "$rootDir/gradle/mkdocs.gradle"
```

Comment:
All the plugins configures are applied at a common place @ Line 33 of build.gradle...

Reply:
The line 33 you have mentioned has plugins required for all projects, we need this plugin only for root project and not all projects.

Reply:
Does the following pattern work ? `gradle.rootProject { apply from: “…” }`
The intent is to ensure that all these config gradle files are applied under a common placeholder to increase readability....